CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1012,7 +1104,11 @@ local function CreateSelector(selectionStringTable, startPosition)
 					this.Selections[this.CurrentIndex].TextTransparency = 0
 				else
 					if GuiService.SelectedCoreObject ~= nil and isAutoSelectButton[GuiService.SelectedCoreObject] then
-						GuiService.SelectedCoreObject = this.SelectorFrame
+						if UserInputService.VREnabled and fixSettingsMenuVR then
+							this.Selections[this.CurrentIndex].TextTransparency = 0
```

Comment:
Do we need to restore the texttransparency here when we disable VR if we ever do in the future?